### PR TITLE
Set the debugoutput channel to error_log instead of echoing it

### LINKED ISSUE #13482: Cannot configure email
I have Owncloud 7.0.4.

When I try to send a test mail to my email (after configuration), I get this error:
Cannot modify header information - headers already sent by (output started at /var/www/3rdparty/phpmailer/phpmailer/class.smtp.php:181) at /var/www/lib/private/appframework/app.php

### PR DESCRIPTION
**Pull request by nickvergessen**
Fix #13482 

@DeepDiver1975

### CODE REVIEW

File: lib/private/mail.php [hunk 1]
```diff
@@ -61,6 +61,9 @@ public static function send($toaddress, $toname, $subject, $mailtext, $fromaddre
 		$mailo->Port = $SMTPPORT;
 		$mailo->SMTPAuth = $SMTPAUTH;
 		$mailo->SMTPDebug = $SMTPDEBUG;
+		if ($mailo->SMTPDebug) {
```
Comment (DeepDiver1975) [inline lib/private/mail.php]: I'd set this even without the if condition

**Comment by DeepDiver1975** [comment]
@karlitschek backport to stable7 requested - THX

**Comment by karlitschek** [comment]
yes. please backport

**Comment by nickvergessen** [comment]
Will make a new PR because of jenkins

**Comment by nickvergessen** [comment]
New PR: #13554

**Pull request closed without merging**